perllib/FixMyStreet/App/Form/Waste/AboutYou.pm

```diff
@@ -62,6 +62,14 @@ has_field email => (
     },
 );
 
+has_field extra_bulky_text_updates => (
```

Comment:
The wording here isn't very clear, what about something like:
<img width="590" alt="image" src="https://github.com/mysociety/fixmystreet/assets/4776/2f33b2c4-7a87-447c-a2d6-5fbab41ca2e1">